fastlane/Fastfile

```diff
@@ -768,17 +768,14 @@ platform :ios do
               # copy docs generated in the previous step into the docs folder
               # and push the changes
               docs_destination_folder = "docs/#{version_number}"
-              index_destination_path = "docs/index.html"
-              migration_guide_destination_path = "docs/v4_api_migration_guide.html"
               FileUtils.cp_r docs_generation_folder + "/.", docs_destination_folder
-              FileUtils.cp docs_index_path, index_destination_path
-              FileUtils.cp docs_migration_guide_path, migration_guide_destination_path
+              docs_files.each do |file|
+                FileUtils.cp file, "docs/"
+              end
```

Comment:
Added a new file here so toop the opportunity to generalize this to we don't need to add new files manually.